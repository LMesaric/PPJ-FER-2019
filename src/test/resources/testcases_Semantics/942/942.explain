semantic: not declared
note: 'retun' on line 2
