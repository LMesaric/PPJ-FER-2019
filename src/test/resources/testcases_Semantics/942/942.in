<prijevodna_jedinica>
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_INT 1 int
   IDN 1 main
   L_ZAGRADA 1 (
   <lista_parametara>
    <lista_parametara>
     <deklaracija_parametra>
      <ime_tipa>
       <specifikator_tipa>
        KR_INT 1 int
      IDN 1 a
    ZAREZ 1 ,
    <deklaracija_parametra>
     <ime_tipa>
      <specifikator_tipa>
       KR_INT 1 int
     IDN 1 b
   D_ZAGRADA 1 )
   <slozena_naredba>
    L_VIT_ZAGRADA 1 {
    <lista_naredbi>
     <naredba>
      <izraz_naredba>
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      IDN 2 retun
       TOCKAZAREZ 2 ;
    D_VIT_ZAGRADA 3 }
