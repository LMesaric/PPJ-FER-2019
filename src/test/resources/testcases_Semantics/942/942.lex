KR_INT 1 int
IDN 1 main
L_ZAGRADA 1 (
KR_INT 1 int
IDN 1 a
ZAREZ 1 ,
KR_INT 1 int
IDN 1 b
D_ZAGRADA 1 )
L_VIT_ZAGRADA 1 {
IDN 2 retun
BROJ 2 0
TOCKAZAREZ 2 ;
D_VIT_ZAGRADA 3 }
