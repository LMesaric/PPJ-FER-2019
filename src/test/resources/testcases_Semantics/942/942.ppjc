int main(int a, int b) {
  retun 0;
}