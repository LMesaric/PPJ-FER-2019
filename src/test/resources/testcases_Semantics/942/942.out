<primarni_izraz> ::= IDN(2,retun)
